content/applications/inventory_and_mrp/manufacturing/workflows/work_center_time_off.rst

```diff
@@ -62,59 +58,57 @@ second work center when the first is unavailable for any reason.
 Add time off for a work center
 ==============================
 
-With configuration completed, time off can now be assigned to the work center that will be made
-unavailable. Begin by navigating to :menuselection:`Manufacturing --> Configuration --> Work
-Centers` and selecting the affected work center. Click :guilabel:`Edit`, and then the :guilabel:`↗
-(external link)` button next to the :guilabel:`Working Hours` drop-down menu.
+With configuration completed, time off can now be assigned to a work center. Begin by navigating to
+:menuselection:`Manufacturing app --> Configuration --> Work Centers`, and selecting the affected
+work center. Click the :icon:`oi-arrow-right` :guilabel:`Internal link` button on the right side of
```

Comment:
```suggestion
work center. Click the :icon:`oi-arrow-right` :guilabel:`(Internal link)` button on the right side of
```
OPTIONAL: suggesting the use of parentheticals here because, since the name of the arrow icon is *technically* 'Internal Link' -- but it **only** appears on the tooltip -- I think the use of parentheticals might be preferred.